fenix/app/src/main/java/org/mozilla/fenix/compose/tabstray/TabGridItem.kt

```diff
@@ -94,95 +99,115 @@ fun TabGridItem(
         Modifier
     }
 
-    Box(
-        modifier = Modifier
-            .wrapContentHeight()
-            .wrapContentWidth(),
+    val dismissState = rememberDismissState(
+        confirmStateChange = { dismissValue ->
+            if (dismissValue == DismissValue.DismissedToEnd || dismissValue == DismissValue.DismissedToStart) {
+                onCloseClick(tab)
+                true
+            } else {
+                false
+            }
+        },
+    )
+
+    SwipeToDismiss(
+        state = dismissState,
+        enabled = !multiSelectionEnabled,
+        backgroundContent = {},
```

Comment:
if you foresee this being unused in more than one location long term, we could make `backgroundContent` optional with `{}` as the default value.

Reply:
I don't think having the background content as empty will be a common use case, let's keep it as non-optional.